torch/quantization/quantize.py

```diff
@@ -88,17 +88,17 @@ def add_observer(module):
             forward_hooks
     """
     for child in module.children():
-        add_observer(child)
+        if type(child) == nnq.FloatFunctional:
+            child.observer = child.qconfig.activation()
+        else:
+            add_observer(child)
 
     # Insert observers only for leaf nodes, note that this observer is for
     # the output of the module, for input QuantStub will observe them
     if hasattr(module, 'qconfig') and module.qconfig is not None and \
        len(module._modules) == 0:
```

Comment:
I think you also need to filter the nnq.FloatFunctional in the condition, since it has more than 0 submodules.

Reply:
it's taken care of in line 91.